src/pudl/workspace/setup.py

```diff
@@ -1,5 +1,5 @@
 """Tools for setting up and managing PUDL workspaces."""
-import importlib
+import importlib.resources as ir
```

Comment:
Is this a common alias for `importlib.resources`?

Reply:
No. ☹️ 

`importlib.resources` is sooo looong, but the method names are so short and generic that they collide with other things `files` and `as_file`. I guess I should really just use the full path everywhere for clarity.